src/foam/java/refinements.js

```diff
@@ -930,8 +930,17 @@ foam.CLASS({
   properties: [
     ['javaType', 'java.util.Map'],
     ['javaJSONParser', 'new foam.lib.json.MapParser()'],
-    ['javaInfoType', 'foam.core.AbstractObjectPropertyInfo'],
+    ['javaInfoType', 'foam.core.AbstractMapPropertyInfo'],
     ['javaFactory', 'return new java.util.HashMap();']
+  ],
+
+  methods: [
```

Comment:
Could we remove this whole block and in properties: above just do:

 [ 'compare', 'return super.compare(o1, o2);' ],

?

Reply:
did not work